app/models/poll.rb

```diff
@@ -12,4 +12,18 @@ def only_one_poll_active
       errors.add(:only_one_poll_can_be_active, "There can be only one poll active for one lecture.")
     end
   end
+
+  def gather_vote_results
+    # reset number of votes for options
```

Comment:
again with the comment block - why not extract it an delete the comment?

Reply:
Done ^.^